tvOS/MALPosterView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  MALPosterView.swift
+//  MyAnimeList (tvOS)
+//
+//  Created by HG on 2020/12/12.
+//
+
+import SwiftUI
+import JikanSwift
+
+struct MALPosterView: View {
+    @State var isShowingDetailView = false
+    // https://stackoverflow.com/a/59196076/4063462
+    @State private var cardButtonFocusd = false
+    
+    let anime: JikanAPIAnime
+    var imageWidth: CGFloat = 200.0
+    var imageHeight: CGFloat {
+        imageWidth * 1.5
+    }
+    private let paddingWhenFocused: CGFloat = 25.0
+    private func carButtonAnimation(isFocused: Bool) -> Animation {
+        return Animation.easeOut(duration: isFocused ? 0.1 : 0.3)
+    }
+    
+    var body: some View {
+        VStack {
+            Button(action: {
+                displayAction()
+            }) {
+                MALPosterItem(isFocusedBinding: $cardButtonFocusd,
+                                imageURL: anime.imageURL)
+                    .frame(width: imageWidth, height: imageHeight)
+            }
+            .buttonStyle(CardButtonStyle())
+            .padding(.all, paddingWhenFocused)
+            .sheet(isPresented: $isShowingDetailView) {
+                AnimeDetailView(anime: anime).environmentObject(store)
+            }
+            Text(anime.title)
+                .lineLimit(1)
```

Comment:
The `.lineLimit(1)` added here causes `EpisodeListView` unable to scroll on my `Apple TV (4th generation) tvOS 14.3 (18K561)`.
On  `Xcode 12.3 (12C33)` with `tvOS Simulators 14.3 (18K559)` it works well.

Not sure if it is a bug on `tvOS 14.3 (18K561)`.

Reply:
I've confirmed no problems on `Apple TV (5th generation) tvOS 14.2 (18K57)`. So it's a bug on `tvOS 14.3 (18K561)`.

Reply:
I upgraded `Apple TV (4th generation)` to `14.4 (18K802)`. The problem is still there.
I also tried DFU restore OS and nothing changes.
It's weird simulators and 5th TV works well.